dss_datamover/utils/config.py

```diff
@@ -57,7 +57,10 @@ def process_config(self):
             config = json.loads(cfg.read().decode('UTF-8', "ignore"))
         if self.params:
             for param in self.params:
-                config[param] = self.params[param]
+                if self.params[param] is not None:  # must explicitly check for None since we still want False values to be overriden onto the config dict
+                    if param == 'compaction' and 'compaction' in config:
```

Comment:
If the compaction is set to **no** in config file and user tries to manually override with **--compaction=yes,** then this logic breaks. Is my understanding correct?